src/Webfactory/Slimdump/SlimdumpCommand.php

```diff
@@ -29,6 +30,13 @@ protected function configure()
                 InputArgument::IS_ARRAY | InputArgument::REQUIRED,
                 'Configuration files (at least one).'
             )
+            ->addOption(
+                'buffer-size',
+                'b',
+                InputOption::VALUE_OPTIONAL,
```

Comment:
I think this should be `VALUE_REQUIRED`? Adding `--buffer-size` without an actual size does not seem to make sense to me.